tests/Date/DateMutabilityConversionTest.php

```diff
@@ -10,10 +10,12 @@
  * @link          http://cakephp.org CakePHP(tm) Project
  * @license       http://www.opensource.org/licenses/mit-license.php MIT License
  */
+namespace Cake\Chronos\Test\Date;
+
 use Cake\Chronos\Date;
 use Cake\Chronos\MutableDate;
 
-class DateMutabilityConversionTest extends TestCase
+class DateMutabilityConversionTest extends \TestCase
```

Comment:
Maybe we should put the date to one a bit more away from the end of the month :)
Or fix it to be non-overflow.